src/zui/ZUIAccessList/index.tsx

```diff
@@ -36,9 +38,7 @@ const ZUIAccessList: FC<ZUIAccessListProps> = ({
           <>
             {showDivider && <Divider />}
             <AccessListItem
-              action={
-                <FormattedMessage id={`zui.accessList.roles.${item.role}`} />
-              }
+              action={<Msg id={messageIds.global.roles[item.role]} />}
```

Comment:
Example of the new `global` map that is always included in the map returned by `makeMessages()`